### ISSUE TITLE
`eslint-disable-line` causes line `max-len`

### ISSUE THREAD

**Issue by Zorgatone** [eslint/eslint#4468]
```JavaScript
import CommonService from "./service/commons"; // eslint-disable-line no-unused-vars
```
```
ESLint Warning max-len Line 1 exceeds the maximum line length of 80. at line 1 col 1
```
Could we add another comment like this, to disable the next line?
```JavaScript
// eslint-disable-next-line no-unused-vars
import CommonService from "./service/commons";
```

My `ESLint` version is `v1.9.0`.

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
You can already do this:

```js
/* eslint-disable no-unused-vars */
import CommonService from "./service/commons";
/* eslint-enable no-unused-vars */
```

**Comment by Zorgatone** [comment]
@nzakas That requires two lines, but ok... I thought that wasy was cleaner

**Comment by michaelficarra** [comment]
You can also do

```js
import "./service/commons";
```

**Comment by Zorgatone** [comment]
@michaelficarra that's not the issue here. I'm using the variable afterwards. I was trying to keep the code not longer that 80 columns, and putting the comment there was causing the rule to trigger.

I also think it's cleaner to have one comment before the line that you want to ignore, instead of inlined

**Comment by michaelficarra** [comment]
If you were actually using the variable, you wouldn't have to disable no-unused-vars. If you don't disable the rule, you don't have the comment. If you don't have the comment, you don't exceed the max line length.

Your claims are inconsistent.

**Comment by Zorgatone** [comment]
The code here is an example
